src/transformed_distribution.jl

```diff
@@ -101,8 +100,8 @@ end
 ##############################
 
 # size
-Base.length(td::Transformed) = length(td.dist)
-Base.size(td::Transformed) = size(td.dist)
+Base.length(td::Transformed) = only(output_size(td.transform, size(td.dist)))
```

Comment:
Based on the docstring here https://github.com/JuliaStats/Distributions.jl/blob/625104475541b6774c3a452facae7e7a989c0fdc/src/matrixvariates.jl#L18 this should be
```suggestion
Base.length(td::Transformed) = prod(output_size(td.transform, size(td.dist)))
```